packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.test.tsx

```diff
@@ -570,4 +572,40 @@ describe('<DateRangeCalendar />', () => {
 
     expect(screen.getAllByMuiTest('pickers-calendar')).to.have.length(3);
   });
+
+  describe('Performances', () => {
```

Comment:
It's interesting putting rendering performance tests here, but oh well, it might help us stop and think more when doing future changes. 💪 
Nitpick, I don't think that such plural form of `Performances` exist. Maybe stick to just `Performance`?

Reply:
I am not a huge fan of those tests
But I don't know how to make better tests without spending a ton of time on them
I think testing the amount of renders is fine here (even though most of the time we prefer to test the render time),
because the regression we want to avoid is the "render all days".

But clearly, they could be improved.

Reply:
Yeah, by no means—they are a way better precaution of re-introducing previous performance issues than nothing. 👍 
And if we get the time or think of better way to implement those—that can be done in future.

Reply:
Named fixed